styles/globals.scss

```
@@ -26,8 +26,9 @@ html,
 body {
   padding: 0;
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, Segoe UI, Roboto, Oxygen,
-    Ubuntu, Cantarell, Fira Sans, Droid Sans, Helvetica Neue, sans-serif;
+  font-family: var(--font-metropolis), -apple-system, BlinkMacSystemFont,
+    Segoe UI, Roboto, Helvetica Neue, sans-serif;
```

Stylelint (17.14.0)

[error] Expected quotes around "Segoe UI" (font-family-name-quotes)

(font-family-name-quotes)